Ports/halflife/package.sh

```diff
@@ -0,0 +1,65 @@
+#!/usr/bin/env -S bash ../.port_include.sh
+port="halflife"
+version="1.0.0"
+useconfigure="true"
+depends=("SDL2")
+workdir="xash3d-fwgs-master"
+files="https://github.com/SerenityPorts/xash3d-fwgs/archive/master.tar.gz xash3d_engine.tar.gz
+https://github.com/SerenityPorts/hlsdk-xash3d/archive/master.tar.gz xash3d_hldll.tar.gz"
+launcher_name="Half-Life"
+launcher_category="Games"
+launcher_command="sh /home/anon/Games/halflife/hl.sh"
+
+# This one is a bit tricky to build, so I'm going a little bit off the script....
+configure() {
+    # Configure the shared object projects (client and game)
+    cd ./hlsdk-xash3d-master
+    ./waf configure -T release
+    cd ../
```

Comment:
We have `run` for that:

https://github.com/SerenityOS/serenity/blob/876424923af03443ed7a3d8c3268051daeb81e0f/Ports/.port_include.sh#L86-L89

Reply:
Can I invoke this on _two_ `workdir`s? This port is (sadly) split into two, as the engine and "game" are split into two different codebases.

Reply:
oh, missed that. Better to keep the explicit `cd`'s then I guess (instead of using `run` for one and `cd` for the other)